src/lib/operators/validate.cpp

```diff
@@ -198,7 +198,7 @@ void Validate::_validate_chunks(const std::shared_ptr<const Table>& in_table, co
           RowIDPosList temp_pos_list;
           temp_pos_list.guarantee_single_chunk();
           for (auto row_id : *pos_list_in) {
-            if (opossum::is_row_visible(our_tid, snapshot_commit_id, row_id.chunk_offset, *mvcc_data)) {
```

Comment:
Is the namespace required here?

Reply:
The function in the anonymous namespace has another parameter list. I would expect we don't need it.

Reply:
Unfortunately not, the compiler throws the following error:
`error: too few arguments to function call, expected 5, have 4; did you mean '::hyrise::is_row_visible'?`

Reply:
Okay.